user-interface/src/main/java/life/qbic/datamanager/files/export/FileNameFormatter.java

```diff
@@ -1,5 +1,6 @@
 package life.qbic.datamanager.files.export;
 
+import java.time.Instant;
```

Comment:
Why the import of `java.time.Instant`? I couldn't find the usage of this import.